static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -233,4 +240,24 @@ function useReplayData({
   };
 }
 
+// see https://github.com/getsentry/sentry/pull/47859
+// replays can apply to many projects when incorporating backend errors
+// this makes having project in the `replaySlug` obsolete
+// we must keep this url schema for now for backward compat but we should remove it at some point
+// TODO: remove support for projectSlug in replay url?
```

Comment:
Set a calendar invite for 90 days when current replays will be out of retention?